_posts/2021-04-09-springboot-autoconfigurations.md

```diff
@@ -0,0 +1,436 @@
+# How Spring Boot's AutoConfigurations Internally Work
```

Comment:
no need for header level 1 here, because that will be generated from the title in the Yaml header. Please copy a header from another article and add it here.